### ISSUE TITLE
Is it possible to store all epochs instead of just evert Xth?

### ISSUE THREAD

**Issue by mstreit** [Caleydo/malevo#100]
I assume that this would result in smoother curves. Maybe we also miss something in the oscillation if we don't store all of them.

@gfrogat, would this be possible? Or is that a storage/runtime problem?

**Comment by gfrogat** [comment]
Might get very large for many runs. Also the database size scales with the number of instances.

 I’ll generate a complete database in the next days and report the numbers.

**Comment by gfrogat** [comment]
One possibility is to quantize the softmax probabilities to Integers [0-255]. Error is relatively small ~ 0.004 and we're quantizing it anyway when generating the softmax heatmaps. Was able to reduce the size of a test database from 240 to 80 Megabytes.

Quantizing is fine as long as we don't start to compute new stuff from the softmax probabilities (e.g. (cross)entropy).

If we store a complete run we're over 2 GB.

It really depends on what we need and want to do

**Comment by thinkh** [comment]
For recent runs @HendrikStrobelt and @gfrogat will store all epochs.

Therefore I close this issue. Please open a new issue, if there is a specific request.
